package.json

```diff
@@ -34,24 +30,23 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "audit-ci": "^4.0.0",
```

Comment:
Any reason why you removed `audit-ci`? If it is not needed, does we need to remove the `audit:ci` script in `packages/itwinui-css/packages.json`?

Reply:
i removed it because it was not being used anywhere. removed the script in next push as well